Tools/autotest/sim_vehicle.py

```diff
@@ -638,8 +638,40 @@ def start_mavproxy(opts, stuff):
     if "extra_mavlink_cmds" in stuff:
         extra_cmd += " " + stuff["extra_mavlink_cmds"]
 
+    # Parsing the arguments to pass to mavproxy
+    # Has been tested by calling:
+    # sim_vehicle.py -m "--state-basedir /dir/dir/dir/ --cmd=\"param load $v2param; module load map\" --master=/dev/ttyUSB0 --console --map --out=udp:192.168.1.1:14550"
```

Comment:
This comment should be remove in favor of only a descriptive comments : something like # Parsing the arguments to pass to mavproxy, split args on space and "=" signs something like that